### PR TITLE
Update UG for add_event

### PR DESCRIPTION
**Pull request by vijay-shankaranand**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -204,6 +204,21 @@ Examples:
...
   all contacts tagged by the tag: customer in JobFestGo.
 
+
+### Adding an event: `add_event`
+
+Adds an event to JobFestGo.
+
+Format: `add_event n/NAME d/DATE a/ADDRESS`
+
+* An event must have mandatory fields name, date and address
```
Comment (tjch-o) [inline docs/UserGuide.md:214]: I think it would be better to add a `:` after the fields

File: docs/UserGuide.md [hunk 2]
```diff
@@ -204,6 +204,21 @@ Examples:
...
+* The event name **must not already exist** in JobFestGo.
+* Date must be valid should be in the appropriate (YYYY-MM-DD) format
```
Comment (tjch-o) [inline docs/UserGuide.md:216]: perhaps `Date should be in the appropriate format: YYYY-MM-DD` is enough or a proper sentence: `Date must be valid and should be in the appropriate (YYYY-MM-DD) format`

**Review by tjch-o — CHANGES_REQUESTED** [review]
lgtm! just some minor nits need fixing

File: docs/UserGuide.md (hunk 1 above)
Comment (vijay-shankaranand) [reply docs/UserGuide.md]: fixed

File: docs/UserGuide.md (hunk 2 above)
Comment (vijay-shankaranand) [reply docs/UserGuide.md]: fixed

**Pull request merged**